android/app/src/main/python/mmrelay/plugins/map_plugin.py

```
@@ -0,0 +1,353 @@
+        renderer.draw().line(path, fill=(255, 0, 0, 255))
+        renderer.draw().text(
+            (x - tw / 2, y - self._arrow - h / 2 - th / 2),
+            self._text,
+            fill=(0, 0, 0, 255),
+        )
+
+    def render_cairo(self, renderer: staticmaps.CairoRenderer) -> None:
+        x, y = renderer.transformer().ll2pixel(self.latlng())
+
+        ctx = renderer.context()
+        ctx.select_font_face("Sans", cairo.FONT_SLANT_NORMAL, cairo.FONT_WEIGHT_NORMAL)
```

Ruff (0.12.2)

Undefined name `cairo`

(F821)